lib/engines/hcktest/hcktest.rb

```diff
@@ -86,10 +86,15 @@ def read_driver(driver)
       Models::Driver.from_json_file(driver_json, @logger)
     end
 
+    def driver_installed?(driver)
+      driver.device == @project.options.test.boot_device
+    end
```

Comment:
Let's merge this method into `find_drivers`; this method is now just one statement.